### ISSUE TITLE
Use dictionary to generate right naming

### ISSUE THREAD

**Issue by mtojek** [elastic/package-registry#264]
At the moment the "import-beats" script uses `strings.Title()` function, so every product name follows the spelling: Activemq, Ibmmq, Coredns, etc.

Implement a mechanism that will adjust all strings, and regenerate integrations.

**Comment by ruflin** [comment]
I wonder if there is a field this could be read out from in Beats?

**Comment by mtojek** [comment]
I would say that it's also messy in Beats (see: statsd).

**Comment by ruflin** [comment]
yes, also in Beats it is not 100% correct. But we should probably fix it there first?

**Comment by mtojek** [comment]
Well, it all depends where you'd like to achieve correctness.

Having a mechanism on the "import-side" will guarantee that if anyone commits an invalid spelling to Beats, these changes won't be propagated to EPR.
Speaking about correcting Beats - I'm fine with this option too, but fixing fields/docs manually will take some time.

/cc @andresrc @dedemorton

**Comment by ruflin** [comment]
For now I would just take what we have in Beats. As soon as we fix it in Beats, it will also propagate to the packages. Also seeing it in the packages wrong will probably push us to fix the modules too. Fixing it in the modules should not rely on you but the team.

**Comment by exekias** [comment]
Having these names in the registry will make these glitches more obvious, I think it's important to know what fields are used from the beats repo to generate packages, so we know where we can fix this in beats so the change gets propagated.

It would be nice to have a small readme explaining how import-beats reads this info

**Comment by mtojek** [comment]
I will go with `fields.yml`: Title.  If you think this is sufficient, you can close the issue.
